sphinx/addnodes.py

```diff
@@ -549,7 +549,7 @@ class literal_strong(nodes.strong, not_smartquotable):
     """
 
 
-class manpage(nodes.Inline, nodes.FixedTextElement):
+class manpage(nodes.reference, nodes.FixedTextElement):
```

Comment:
I am not sure if this change is really needed.

Reply:
> I think changing the base class of `manpage` is not the way to go. Concerning the formatting, I think we shouldn't change it for now.

@picnixz: I tried reverting this change but I get the same formatting results in HTML. It must come from something else.
We can also wait for another opinion on the formatting changes, and if it is OK I can modify the expected results in the tests, and add a changelog entry etc.

Otherwise I don't really know where to look for this formatting change in the code. I will continue to try some things but I don't know if I will manage to keep the existing HTML formatting.

Reply:
Would it be ok for you to postpone it for a few weeks/months? because I won't have time for in-depth review before march/april. 

If anyone else has time to dig or wants to discuss feel free to ignore my opinion as I don't mind both formats actually (just that I want to keep changes minimal if possible, especially if it concerns base classes)